### ISSUE TITLE
Gnome 44: PaperWM keyboard shortcuts clash with gnome default `super+left/right`?

### ISSUE THREAD

**Issue by jtaala** [paperwm/PaperWM#516]
I don't usually use `super+left/right` to switch to left/right window, but on my test setup for Gnome 44 I notice that sometimes when logging in the PaperWM default switch to left/right sometimes doesn't work - or on `super+left` it moves the window.

I've disabled the default `Super+left/right` gnome shortcut (which tiles the window left/right) and the issue doesn't appear to be happening now.  Not sure if that was the problem.

Anyone else on Gnome 44 also seeing this?

**To Reproduce**
Unfortunately I can't get this to reproduce consistently.   It happened periodically after logging in (didn't seem very consistent).  I also think I saw it happen after a lock-screen.

**Expected behavior**
PaperWM keyboard shortcut should always work.

**System information:**

```
Distribution: Fedora Linux 38
GNOME Shell 44.0
Display server: Wayland
PaperWM branch/tag: fixes-gnome-44-support
PaperWM commit: 56023a79c5f827749de2bd868208570553b60cca
Enabled extensions:
- paperwm@hedning:matrix.org
```

**Comment by Lythenas** [comment]
I think we can just remove the conflicting shortcuts in the `set-recommended-gnome-shell-settings.sh` and document it with the other recommended settings.

**Comment by fumbles** [comment]
My Super + Left/Right work fine but Shift+Super+< does not for me even after disabling everything I can find in the keyboard shortcut stuff.
 I pulled the fixes-gnome-44-support commit into my local develop branch..hence it says develop branch below.


```Distribution: Fedora Linux
GNOME Shell 44.0
Display server: Wayland
PaperWM branch/tag: develop
PaperWM commit: 56023a79c5f827749de2bd868208570553b60cca
Enabled extensions:
- paperwm@hedning:matrix.org
- drive-menu@gnome-shell-extensions.gcampax.github.com
- dash-to-dock@micxgx.gmail.com
- clipboard-indicator@tudmotu.com
- utcclock@injcristianrojas.github.com
- apps-menu@gnome-shell-extensions.gcampax.github.com
- background-logo@fedorahosted.org
- places-menu@gnome-shell-extensions.gcampax.github.com
- appindicatorsupport@rgcjonas.gmail.com
- caffeine@patapon.info
```

**Comment by crispinb** [comment]
I experienced the clash with Gnome's 'view split on Left' & also fixed by disabling the Gnome shortcut. Hadn't noticed until seeing this as I normally use PaperWM's 'switch to the next/previous window'

**Comment by jtaala** [comment]
> I think we can just remove the conflicting shortcuts in the `set-recommended-gnome-shell-settings.sh` and document it with the other recommended settings.

@Lythenas given your skills in bash/sh (the only bash I do is bashing my head against a wall when I try figure it out!) - could you give this one a go when you get time?

**Comment by jtaala** [comment]
Ha, this looks quite similar to #233.

**Comment by jtaala** [comment]
Closing as fixed in #515 which has now been merged into `develop` branch.
